src/services/billing/cost-management/cost-analysis/lib/helper.ts

```diff
@@ -56,6 +56,12 @@ export const getTimeUnitByPeriod = (granularity: GRANULARITY, start: Dayjs, end:
     return 'year';
 };
 
+export const getDateFormatByTimeUnit = (timeUnit: TimeUnit): string => {
+    if (timeUnit === 'month') return 'YYYY-MM';
+    if (timeUnit === 'year') return 'YYYY';
+    return 'YYYY-MM-DD';
+};
```

Comment:
이것도 좋은 방법이지만, 아래 방법을 권장드려요!
```suggestion
{
 month: 'YYYY-MM',
 year: 'YYYY',
 date: 'YYYY-MM-DD',
};
```

Reply:
어디서 이 방식이 좋다고...하는걸 봐서 ㅋㅋ

Reply:
옙!!!!